### ISSUE TITLE
recommendation for program robustness

### ISSUE THREAD

**Issue by Jiexin-Zheng** [yaoing/DAN#17]
hello, thanks for your work, I think you can add the following codes to the definition of PartitionLoss() function for program robustness:

>             eps = sys.float_info.epsilon
>             loss = torch.log(1+num_head/(var + eps))

**Comment by yaoing** [comment]
Thanks! Code has been updated!
